src/core/HTTP/RestClient.ts

```diff
@@ -62,8 +62,11 @@ export default class RestClient {
     this.client = axios.create({
       baseURL: this.clientUrl,
       headers: this.clientHeaders,
-      httpAgent: new HTTPAgent({ keepAlive: true }),
-      maxContentLength: 10 * 1000000, // 10mb is kong file system
+      httpAgent: new HTTPAgent({ keepAlive: true, maxSockets: 10 }),
```

Comment:
why adding the `maxSockets`?

Reply:
Ah that was during my testing, will remove!